examples/dreambooth_custom_character/README.md

```diff
@@ -0,0 +1,200 @@
+# Creating a Cute Custom Character with Stable Diffusion and Dreambooth
+
+## Introduction
+[Stable Diffusion](https://github.com/CompVis/stable-diffusion) is an incredible open-source tool for fast, effective generation of novel images across a wide variety of domains. Despite its power and convenience, it can struggle to create consistent, identifiable characters in specific styles.
+
+While improvements in prompting can mitigate this to some extent, it is almost always a better bet to fine-tune the model with your own data. Here are a few reasons why: 
+
+1. No amount of clever prompting can lead the neural network to generate things it hasn’t seen enough of in its training data. We’re pretty confident there wasn’t a preponderance of fluffy cartoon oxen in [Stable Diffusion’s training data](https://laion.ai/blog/laion-5b/), which leads to the lackluster baseline results we’ll show below. It’s much easier for us to provide this information to the model directly than to wait and hope that a future version of Stable Diffusion will somehow find and include precisely the data we’re looking for.
+2. If you train or fine-tune open source models with your own data rather than remaining beholden to third-party services, you can embed the model directly into your application. This will likely save you money over purchasing credits to existing providers, and eliminate your dependency on a vendor that could change its API / pricing or stop operating at any time.
+3. Fine-tuning with your own proprietary data allows you to gain a competitive advantage that you can’t access with prompting alone. If your IP relies solely on the specific way you prompt a publicly accessible model, it’s much easier for competitors to replicate your results!
+
+At [Oxen](https://oxen.ai/), we’re firm believers in the power of enhancing foundational models with domain-specific data. In this tutorial, we’ll use an Oxen repository and the [Dreambooth](https://dreambooth.github.io/) fine-tuning technique to create a Stable Diffusion variant that can consistently reproduce cute, fluffy cartoon oxen like the one currently hanging out [on our 404 page](https://www.oxen.ai/not-a-real-url).
```

Comment:
I think we should move this above the 1,2,3 reasons why you should fine tune, just to get to the punchline of what we are about to do quicker.